distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -89,8 +88,11 @@ solr.useBasicAuth=true
 solr.username=admin
 
 # Solr Cloud Provider
-# solr.client=CloudSolrClient
-# solr.cloud.zookeeper=zookeeperhost1:2181,zookeeperhost2:2181,zookeeperhost3:2181
+solr.client=CloudSolrClient
+solr.cloud.zookeeper=localhost:2181
+# solr.cloud.shardCount=2
+# solr.cloud.replicationFactor=2
+# solr.cloud.maxShardPerNode=2
```

Comment:
Can we provide any guidance for these properties? Also, is there a reason we do not set the values, but instead comment them out?

Reply:
Looks like I did not need that system property in the final version of the quick start after all.  I can remove these.